README.md

```diff
@@ -1,221 +1,276 @@
+[affects your bottom line](http://devnet.kentico.com/Blogs/Thomas-Robbins/September-2012/How-loading-time-affects-your-bottom-line-Infograp.aspx).
+*Asynchronous* ads do not block the page and can be delivered after core
+content - [Async FTW](http://www.krux.com/blog/krux-engineers/synchronous-versus-asynchronous-tags-whats-the-big-deal/).
 
+Why is it so hard to deliver ads asynchronously? Because they may contain calls to `document.write`, which expects to be handled synchronously.
+**PostScribe lets you deliver a synchronous ad asynchronously without modifying the ad code**.
 
 ### Approach
 
-Other tag writing libraries exist (see [alternatives](#alternatives)), but PostScribe is novel in it's use of what we call DOM Proxies, a way to ensure that the content is written as close to the way the browser would natively write the content with `document.write`/`innerHTML`. Read: it behaves just like the browser would, without convoluted parsing or hacks.
+Other tag writing libraries exist (see [alternatives](#alternatives)), but PostScribe is novel in its use of what we call DOM Proxies, a way to
+ensure that the content is written as close to the way the browser would natively write the content with `document.write`/`innerHTML`. Read: it
+behaves just like the browser would, without convoluted parsing or hacks.
 
 For more information:
 
-* [Presentation at HTML5devconf](http://youtu.be/ClzeilKwX10) by the author, Derek Brans
+* [Presentation at HTML5devconf](http://youtu.be/ClzeilKwX10) by the original author, Derek Brans
 * [Interactive Demo](http://run.plnkr.co/plunks/e8MJAx/) with side by side comparisons of other tag writers
-* [Documentation](https://github.com/krux/postscribe/tree/master/doc)
-* Browse the [raw](https://github.com/krux/postscribe/blob/master/postscribe.js) or [annotated](http://krux.github.com/postscribe/doc/postscribe.html) source code.
-
 
 # Getting Started
 
-PostScribe overrides document.write. It is best and safest to use PostScribe after DOM is ready.
+PostScribe overrides `document.write`. It is best and safest to use PostScribe after DOM is ready.
+
+## Installation
+
+### Browser
+
+If you just want to use the script without installing anything, use the following to load the script from *cdnjs*:
+
+```html
+<script src="https://cdnjs.cloudflare.com/ajax/libs/postscribe/2.0.0/postscribe.min.js"></script>
+```
+
+### NPM
+
+You can include `postscribe` using *npm*:
+
+```console
+npm install --save-dev postscribe
+```
+
+### Bower
+
+You can include `postscribe` using *bower*:
 
-Include `./htmlParser/htmlParser.js` and `./postscribe.js` on your page. TODO: cdn for postscribe.min.js
+```console
+bower install --save-dev postscribe
+```
+
+## Accessing
+
+### ES6/ES2015
+
+```javascript
+import {postscribe} from 'postscribe';
+```
+
+### AMD
+
+```javascript
+define(['postscribe'], function(postscribe) {
+
+});
+```
+
+### CommonJS
+
+```javascript
+var postscribe = require('postscribe');
```

Comment:
If maintaining two repositories, builds and sets of tests isn't too much it's definitely the best option here. It'd allow clean imports of both postscribe and htmlParser.

What would change, however, is that you'd have to generate a 'special' build for CommonJS targets (and publish that as the main/browser script on NPM). Since htmlParser will become a 'real' NPM dependency, bundling it with the CommonJS build would mean it'd be downloaded twice (and you'd have to update postscribe for every new htmlParser version).

Your Webpack build for anything-but-CommonJS would be the same as it is now (for Postscribe). The CommonJS build would look like this, excluding htmlParser from the build and just requiring it instead:

```
export default {
  entry: 'postscribe',
  resolve: {
    root: path.resolve('./src')
  },
  module: {
    loaders: ... // unchanged
  },
  output: {
    filename: 'postscribe.commonjs.js',
    libraryTarget: 'commonjs2'
  },
  externals: ['html-parser'] // assuming the package would be named html-parser
};
```

**Assuming postscribe will depend on something like `html-parser: "^1.0.0"`:**

Advantages:

- Everyone using Webpack or Browserify will have a pleasant, clean experience, being able to easily use htmlParser and postscribe by just directly requiring/importing both instead of using named imports, with and without Babel
- No need to bump postscribe versions for new (compatible) versions of htmlParser, as long as they're compatible nothing will break.

Disadvantages:

- People using Webpack or Browserify would have a hard time importing a specific version of "postscribe+htmlParser". If htmlParser would become a dependency of postscribe they'd have no direct control over the version of htmlParser used. postscribe would use whatever the latest 1.x.x version of htmlParser would be. (The way around this would be depending on a fixed htmlParser version and bumping the postscribe version every time.)
- Users that need both postscribe and direct access to htmlParser without a modern Javascript build system, would effectively include htmlParser twice (once on it's own, once inside the postscribe build). I'm not sure if this happens very often.

With all of this said, I'd still split the projects if it's not too much of a hassle. It puts oldschool-style users that use both postscribe and htmlParser directly at a slight disadvantage, keeps everything the same for current users of postscribe and significantly improves things for Webpack/Browserify/Babel/CommonJS users of postscribe/htmlParser.